backend/middleware/authorisationMiddleware.ts

```diff
@@ -17,6 +18,7 @@ const authorisationMap = {
   '/create-user': [AuthRole.CREATE_USER],
   '/delete/children/none': [AuthRole.MAINTAIN_OAUTH_USERS],
   '/create-external-user': [AuthRole.MAINTAIN_OAUTH_USERS, AuthRole.GROUP_MANAGER],
+  '/create-role': [AuthRole.ROLES_ADMIN],
```

Comment:
We need an additional test in the UI/cypress test to ensure the page goes to the AuthError if unauthorized

Reply:
Agree and added